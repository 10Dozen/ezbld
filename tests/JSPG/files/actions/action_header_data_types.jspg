# Scene

@ Action1 TEXT | dialog_left pic.jpg

@ Action1 VARIABLE | dialog_left `Foo.Bar[0]`

@ Action1 FUNCTION | dialog_left { return Foo.Bar() }


@ Action2 TEXT | dialog_left pic.jpg | MyTag

@ Action2 VARIABLE | dialog_left `Foo.Bar[0]` | MyTag

@ Action2 FUNCTION | dialog_left { return Foo.Bar() } | MyTag


@ Action3 TEXT | dialog_left pic.jpg |

@ Action3 VARIABLE | dialog_left `Foo.Bar[0]` |

@ Action3 FUNCTION | dialog_left { return Foo.Bar() } |