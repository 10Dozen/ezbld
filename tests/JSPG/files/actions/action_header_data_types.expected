[0]
cls = JSPGScene
name = Scene | TEXT

[1]
cls = JSPGAction
name = Action1 TEXT | TEXT
type = dialog_left | TEXT
portrait = pic.jpg | TEXT
scene = Scene | TEXT

[2]
cls = JSPGAction
name = Action1 VARIABLE | TEXT
type = dialog_left | TEXT
portrait = Foo.Bar[0] | VARIABLE
scene = Scene | TEXT

[3]
cls = JSPGAction
name = Action1 FUNCTION | TEXT
type = dialog_left | TEXT
portrait = ()=>{ return Foo.Bar() } | FUNCTION
scene = Scene | TEXT


[4]
cls = JSPGAction
name = Action2 TEXT | TEXT
type = dialog_left | TEXT
portrait = pic.jpg | TEXT
tag = MyTag | TEXT
scene = Scene | TEXT

[5]
cls = JSPGAction
name = Action2 VARIABLE | TEXT
type = dialog_left | TEXT
portrait = Foo.Bar[0] | VARIABLE
tag = MyTag | TEXT
scene = Scene | TEXT

[6]
cls = JSPGAction
name = Action2 FUNCTION | TEXT
type = dialog_left | TEXT
portrait = ()=>{ return Foo.Bar() } | FUNCTION
tag = MyTag | TEXT
scene = Scene | TEXT


[7]
cls = JSPGAction
name = Action3 TEXT | TEXT
type = dialog_left | TEXT
portrait = pic.jpg | TEXT
scene = Scene | TEXT

[8]
cls = JSPGAction
name = Action3 VARIABLE | TEXT
type = dialog_left | TEXT
portrait = Foo.Bar[0] | VARIABLE
scene = Scene | TEXT

[9]
cls = JSPGAction
name = Action3 FUNCTION | TEXT
type = dialog_left | TEXT
portrait = ()=>{ return Foo.Bar() } | FUNCTION
scene = Scene | TEXT
